README.md

```diff
@@ -78,12 +78,12 @@ The following assumes the reader knows basics of kubernetes.
 6. Apply to Kubernetes Cluster: You need to apply all the files in the `kubernetes/` directory to your cluster. Some `ReplicationControllers` are dependant on other components, so it'll be helpful to apply them at last. This is the recommended order:
 
 ```bash
-kubectl apply -f ConfigMap.yml
-kubectl apply -f ClusterIssuer.yml
-kubectl apply -f PersistentVolumeClaim.yml
-kubectl apply -f Service.yml
-kubectl apply -f Ingress.yml
-kubectl apply -f ReplicationController.yml
+kubapply -f ConfigMap.yml
```

Comment:
Okay, this is good. Except, let's rename the script to `kubeapply` just because all the kubernetes scripts start with `kube`! :smile: